lib/src/utils/markdown.dart

```diff
@@ -218,12 +228,14 @@ String markdown(
     ],
     inlineSyntaxes: [
       StrikethroughSyntax(),
+      MultipleLinebreaksSyntax(),
       SpoilerSyntax(),
       EmoteSyntax(getEmotePacks),
       PillSyntax(),
       MentionSyntax(getMention),
       InlineLatexSyntax(),
     ],
+    inlineOnly: true,
```

Comment:
what is it?

Reply:
It allows syntaxes to process the whole message and not line by line. This way I can know if there are 2 linebreaks back to back